pkgs/development/python-modules/cwcwidth/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, cython }:
+
+buildPythonPackage rec {
+  pname = "cwcwidth";
+  version = "0.1.4";
```

```suggestion
  version = "0.1.4";
  format = "pyproject";
```
https://github.com/sebastinas/cwcwidth/blob/v0.1.4/pyproject.toml

---

What's the advantage over setuptools?

---

pyproject is likely to be the more supported use case moving forward

---

Changed then